components/content/dancing-blob/index.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <TresCanvas :clear-color="'#0c1a30'">
+    <TresPerspectiveCamera :position="[13, 0, 0]" />
+    <OrbitControls />
+    <Suspense>
```

Comment:
You don't need suspense here, I don't see any top level await (the one in the onMounted doesn't count)